main/webapp/modules/core/styles/index/open-project-ui.less

```diff
@@ -185,6 +187,53 @@ ul#tagsUl li a  {
 	-moz-border-radius: 10px;
 	-webkit-border-radius: 10px;
 }
+/* Search bar in open project */
+// search
+#divSearch{
+    float: left;
+    width: 2.5%;
+    margin-left: -0.5%;
+}
+#divImage{
+    float: left;
+    width: 90%;
+}
+
+#divInput{
+    float: right;
+    width: 10%;
+    display: none;
+}
+
+// input
+ #searchInProjects{
+    border-radius: 10px;
+ }
+
+.header-search-box{
+    display: none;
+    width: 0px;
+    float: left;
+}
+
+// icon search
+.magnifying_glass{
+    width: 40px;
+    height: 27px;
+    border-radius: 10px;
+}
+
+.magnifying-glass-open{
+    background-color: #DEE6FF;
+    width: 100%;
+}
+
+#no-results-message{
+    color: red;
+    display: none;
+    text-align: center;
+    font-size: x-large;
+}
```

Comment:
Perhaps this `red` is a tiny bit violent? How about something like `#777` which we already use to display the OpenRefine version number in the bottom left corner?